integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/api/CrowdNodeWebApi.kt

```diff
@@ -42,20 +61,164 @@ interface CrowdNodeWebApi {
         @Path("address") address: String
     ): Response<AddressStatus>
 
+    @GET("odata/apiaddresses/UsingDefaultApiEmail(address='{address}')")
+    suspend fun hasDefaultEmail(
+        @Path("address") address: String
+    ): Response<IsDefaultEmail>
+
     @GET("odata/apimessages/SendMessage(address='{address}',message='{message}',signature='{signature}',messagetype=1)")
     suspend fun sendSignedMessage(
         @Path("address") address: String,
         @Path("message") message: String,
         @Path("signature") signature: String,
     ): Response<MessageStatus>
 
-    @GET("odata/apiaddresses/UsingDefaultApiEmail(address='{address}')")
-    suspend fun hasDefaultEmail(
-        @Path("address") address: String
-    ): Response<IsDefaultEmail>
-
     @GET("odata/apimessages/GetMessages(address='{address}')")
     suspend fun getMessages(
         @Path("address") address: String
     ): Response<List<MessageStatus>>
 }
+
+@ExperimentalTime
+open class CrowdNodeWebApi @Inject constructor(
+    private val endpoint: CrowdNodeEndpoint,
+    private val analyticsService: AnalyticsService
+) {
+    companion object {
+        private val log = LoggerFactory.getLogger(CrowdNodeWebApi::class.java)
+        private val MAX_PER_TX_KEY = "AmountApiWithdrawalMax"
+        private val MAX_PER_1H_KEY = "AmountApiWithdrawal1hMax"
+        private val MAX_PER_24H_KEY = "AmountApiWithdrawal24hMax"
+    }
+
+    // TODO: these methods are just mappers right now. Move more logic in here from the aggregator class
```

Comment:
`CrowdNodeApiAggregator` is getting quite big. Some of the webAPI-related stuff should be moved to a dedicated class, but not everything can be quickly refactored.